cisagov/gophish-tools
# Remove appended campaign id appended to new campaign name

Remove appended CX id to campaign name when creating new campaign. We are changing the way we parse exports and will reference by the format "_levelX" instead of CX identifiers. This gives us the ability to correlate levels of assessments with campaigns in GoPhish exports.

## 🗣 Description ##

This removes the old structure of appending CX to the campaign name. We no longer need this format and will be referencing by level_id (1-6).

## 💭 Motivation and context ##

We no longer need this format and will be referencing by level_id (1-6).

## 🧪 Testing ##

To test and validate that the name format change was implemented correctly, I did the following:

1. Build a new image that includes the changes in this branch and tag it for reference by running:
`docker build -t gophish-tools-issue-43 .`

2. Run the following to override the default image and reference the newly tagged image with the changes:
`export GOPHISH_TOOLS_IMAGE="gophish-tools-issue-43" && eval "$(docker run gophish-tools-issue-43)"`

3. The alias should be setup to drop into a bash instance on the gophish-tools container. You can run the following to access the container:
`gophish-tools-bash`

4. . In the contaner bash instance, create a new assessment as you normally would by running the alias:
 `pca-wizard`

5. Input Campaign data and import into gophish. The campaign name should now exclude the appended _C1 that was originally there.

**_I also confirmed that the campaign went through all phases correctly sending email and generating click output as expected with the new name format._**



## ✅ Checklist ##
